content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,403 @@
+
+- **Removed** the `ENABLE_AUTO_MTLS_CHECK_POLICIES` feature flag.  
+
+- **Removed** the `PILOT_ENABLE_LEGACY_AUTO_PASSTHROUGH` feature flag.
+  
+
+- **Removed** the `PILOT_ENABLE_LEGACY_ISTIO_MUTUAL_CREDENTIAL_NAME` feature flag.  
+
+- **Removed** the `PILOT_LEGACY_INGRESS_BEHAVIOR` feature flag.  
+
+- **Removed** the `PILOT_ENABLE_ISTIO_TAGS` feature flag.  
+
+- **Removed** the `ENABLE_LEGACY_LB_ALGORITHM_DEFAULT` feature flag.  
+
+- **Removed** the `PILOT_PARTIAL_FULL_PUSHES` feature flag.  
+
+- **Removed** the `PILOT_INBOUND_PROTOCOL_DETECTION_TIMEOUT` feature flag. This can be configured in MeshConfig if needed.
+  
+
+- **Removed** the `AUTO_RELOAD_PLUGIN_CERTS` feature flag.
+  
+
+- **Removed** the `PRIORITIZED_LEADER_ELECTION` feature flag.
+  
+
+- **Removed** the `SIDECAR_IGNORE_PORT_IN_HOST_MATCH` feature flag.
+  
+
+- **Removed** the `REWRITE_TCP_PROBES` feature flag.
+  
+
+- **Removed** support for XDS v2 types in `EnvoyFilter`s. These should use the v3 interface. This has been a warning for multiple releases and is now upgraded to an error.
+  
+
+- **Removed** the `PILOT_ENABLE_PROTOCOL_SNIFFING_FOR_OUTBOUND` and `PILOT_ENABLE_PROTOCOL_SNIFFING_FOR_INBOUND` feature flags. These have been enabled by default since Istio 1.5.
+  
+
+- **Removed** support for looking up Envoy extensions in `EnvoyFilter` configuration by name without the typed config URL.
+  
+
+- **Optimized** EnvoyFilter index generation to avoid rebuilding all EnvoyFilters everytime a single one has changed, instead only rebuild the changed EnvoyFilter and update it in place.
+  
+
+
+## Security
+
+
+
+
+- **Added** `insecureSkipVerify` implementation from DestinationRule. Setting `insecureSkipVerify` to `true` will disable CA certificate and Subject Alternative Names verification for the host.
+  ([Issue #33472](https://github.com/istio/istio/issues/33472))
+
+- **Added** support for PeerAuthentication policies in Ambient
+  ([Issue #42696](https://github.com/istio/istio/issues/42696))
+
+- **Added** cipher_suites support for non ISTIO_MUTUAL traffic through MeshConfig API.
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Added** cipher_suites support for mesh to mesh traffic through MeshConfig API.
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Added** Certificate Revocation List(CRL) support for peer certificate validation.
+  
+
+- **Added** support for a flag called USE_EXTERNAL_WORKLOAD_SDS, when set to true, it will require an external SDS workload socket and it will prevent the istio-proxy from starting if the workload SDS socket is not found.
+  ([Issue #45534](https://github.com/istio/istio/issues/45534))
+
+
+- **Fixed** use defer to unlock mutex
+  
+
+- **Fixed** an issue where jwk issuer was not resolved correctly when having a trailing slash in the issuer URL.
+  ([Issue #45546](https://github.com/istio/istio/issues/45546))
+
+
+- **Removed** the `SPIFFE_BUNDLE_ENDPOINTS` feature flag.
+  
+
```

Comment:
nit: should these empty lines be removed or are they ignored when rendered?

Reply:
They are in the process of being removed. This PR is still a WIP but reviews on content are welcome